pkgs/development/compilers/llvm/3.9/llvm.nix

```diff
@@ -39,7 +42,13 @@ in stdenv.mkDerivation rec {
 
   outputs = [ "out" ] ++ stdenv.lib.optional enableSharedLibraries "lib";
 
-  buildInputs = [ perl groff cmake libxml2 python libffi ]
+  buildInputs = [
+    buildPackages.perl
```

Comment:
Can these be `nativeBuildInputs`? might not even cause rebuild for native due to some normalizing in stdenv.